### ISSUE TITLE
Weird error ssl related

### ISSUE THREAD

**Issue by Ironjanowar** [rockneurotiko/ex_gram#19]
```
2019-03-29T17:30:45.320Z [error] GenServer #PID<0.295.0> terminating
** (FunctionClauseError) no function clause matching in ExGram.Updates.Polling.handle_info/2
    (ex_gram) lib/ex_gram/updates/polling.ex:21: ExGram.Updates.Polling.handle_info({:ssl_closed, {:sslsocket, {:gen_tcp, #Port<0.8>, :tls_connection, :undefined}, [#PID<0.292.0>, #PID<0.291.0>]}}, {:spotify_uri_bot, "AWESOME_TOKEN"})
    (stdlib) gen_server.erl:637: :gen_server.try_dispatch/4
    (stdlib) gen_server.erl:711: :gen_server.handle_msg/6
    (stdlib) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
Last message: {:ssl_closed, {:sslsocket, {:gen_tcp, #Port<0.8>, :tls_connection, :undefined}, [#PID<0.292.0>, #PID<0.291.0>]}}
State: {:spotify_uri_bot, "AWESOME_TOKEN"}
```

Happened to mi on this ![bot](https://github.com/Ironjanowar/spotify_uri_bot)

**Comment by Ironjanowar** [comment]
Happens every 5 minutes (more or less)
```
2019-03-29T17:30:45.320Z [error] GenServer #PID<0.295.0> terminating
** (FunctionClauseError) no function clause matching in ExGram.Updates.Polling.handle_info/2 ...

2019-03-29T17:35:59.029Z [error] GenServer #PID<0.303.0> terminating
** (FunctionClauseError) no function clause matching in ExGram.Updates.Polling.handle_info/2 ...

2019-03-29T17:41:12.740Z [error] GenServer #PID<0.311.0> terminating
** (FunctionClauseError) no function clause matching in ExGram.Updates.Polling.handle_info/2 ...
```

**Comment by rockneurotiko** [comment]
#wontfixever

**Comment by rockneurotiko** [comment]
Joke, this was fixed in #22
